tests/_Failure/tests/test.py

```diff
@@ -18,19 +16,6 @@ def tearDown(self):
             self.view.window().focus_view(self.view)
             self.view.window().run_command("close_file")
 
-    # since ST2 doesn't support unittest.skip, we have to do primitive skipping
-    if version < '3000':
-        def test_hello_world_2(self):
-            self.view.run_command("insert", {"characters": "hello world"})
-            first_row = self.view.substr(self.view.line(0))
-            self.assertEqual(first_row, "hello world")
-
-    if version >= '3000':
```

Comment:
You shouldn’t remove this block

Reply:
Sure, but what are they there for?

Reply:
Oh, i now see that you removed them because they are redundant. Ignore my preview comments.